packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -62,6 +62,77 @@ export class Relayer {
     this.l2BlockFinder = new BlockFinder<BlockTransactionBase>(this.l2Client.l2Web3.eth.getBlock);
   }
 
+  // Evaluates given `relayableDeposit` for the `l1Token` and determines whether to submit a slow or fast relay.
+  private async _relayPendingDeposit(l1Token: string, relayableDeposit: RelayableDeposit): Promise<void> {
```

Comment:
this is pretty clean! I really like the logic flow this enables in `checkForPendingDepositsAndRelay`.

Reply:
one small nit: can you put this method down below `checkForPendingDepositsAndRelay`? ideally all "public" functions should come before the "private" ones.

Reply:
Agreed! easier to reason about and maintain to with this refactor